FRAuth/FRAuth/Authentication/AuthService.swift

```diff
@@ -229,7 +229,9 @@ public class AuthService: NSObject {
                     }
                 }
                 else if let tokenId = response[OpenAM.tokenId] as? String {
```

Comment:
Does the user has a was way to get the failureURL on any Exception.

Reply:
@witrisna Do you have an answer for this?

Reply:
When failed with the Journey, Android throws AuthenticationException with the raw body result, the raw body should contains the failureURL.